include/fmt/std.h

```diff
@@ -435,6 +437,28 @@ struct formatter<BitRef, Char,
   }
 };
 
-FMT_END_NAMESPACE
+FMT_EXPORT
+template <typename T, typename Char>
+struct formatter<std::atomic<T>, Char> : formatter<T, Char> {
+  template <typename FormatContext>
+  auto format(const std::atomic<T>& v, FormatContext& ctx) const
+      -> decltype(ctx.out()) {
+    return formatter<T, Char>::format(v.load(), ctx);
+  }
+};
 
+FMT_EXPORT
+template <typename Char>
+struct formatter<std::errc, Char>
+    : formatter<std::underlying_type<std::errc>::type, Char> {
+  typedef std::underlying_type<std::errc>::type underlying_type;
```

Comment:
I suggest move ``underlying_type`` into ``format`` function and replace ``typedef`` with
```c++
using underlying_type = std::underlying_type<std::errc>::type;
```

Reply:
Fixed